pkg/server/router/manifest.go

```diff
@@ -182,3 +184,242 @@ func (mr ManifestRouter) DeleteManifest(ctx context.Context, w http.ResponseWrit
 
 	return framework.Respond(ctx, w, nil, http.StatusOK)
 }
+
+type SubmitApplicationRequest struct {
+	ManifestID             string                              `json:"manifestId" validate:"required"`
```

Comment:
why not use https://github.com/TBD54566975/ssi-sdk/blob/main/credential/manifest/model.go#L81 directly?

Reply:
like
```go
type SubmitApplicationRequest struct {
   Credential Application ssisdk.CredentialApplication `json:"credential_application" validate:"required"`
}
```

Reply:
Right, I was following the Credential.go example where it went one level into the object and created fields for them.

I could definitly be convinced to just have the global CredentialApplication ssisdk.CredentialApplication object, but it would also require the user to submit their own ID (which we generate with guid)

Also setting up top level manifestId and could perhaps allow for better querying capability with our db implementation (or not)

Reply:
I do imagine that a compliant wallet will generate a fully formed and compliant Credential Application.

I do not think there is necessarily a problem with a user generating their own identifier. We could generate our own and map to theirs if we are worried about a conflict, though it should be fine given the spec says:
> The value of this property MUST be a unique identifier, such as a UUID.

Reply:
Ok that sounds good, makes it more standardized on our end as well. We almost have a single source of truth for data format in SSI-SDK.

I'll make changes for this